ios/connect.go

```diff
@@ -333,3 +362,39 @@ func initializeXpcConnection(h *http.HttpConnection) error {
 
 	return nil
 }
+
+func ConnectUserSpaceTunnel(remoteIp string, port int) (*net.TCPConn, error) {
+	addr, _ := net.ResolveTCPAddr("tcp4", fmt.Sprintf("localhost:%d", DefaultTunnelPort()))
+	conn, err := net.DialTCP("tcp", nil, addr)
+	if err != nil {
+		return nil, fmt.Errorf("ConnectUserSpaceTunnel: failed to dial: %w", err)
+	}
+	_, err = conn.Write(net.ParseIP(remoteIp).To16())
+	portBytes := make([]byte, 4)
+	binary.LittleEndian.PutUint32(portBytes, uint32(port))
+	_, err1 := conn.Write(portBytes)
+	return conn, errors.Join(err, err1)
+}
+
+// defaultHttpApiPort is the port on which we start the HTTP-Server for exposing started tunnels
+// 60-105 is leetspeek for go-ios :-D
+const defaultHttpApiPort = 60105
+
+// DefaultHttpApiPort is the port on which we start the HTTP-Server for exposing started tunnels
+// if GO_IOS_AGENT_PORT is set, we use that port. Otherwise we use the default port 60106.
+// 60-105 is leetspeek for go-ios :-D
+func DefaultHttpApiPort() int {
```

Comment:
```suggestion
func HttpApiPort() int {
```
it's only the default if no environment variable is set